lib/braveAlerter.js

```diff
@@ -279,19 +279,28 @@ class BraveAlerter {
 
       // Store the results and idemopotently get the respondedByPhoneNumber for the session
       const changedAlertSession = new AlertSession(alertSession.sessionId, nextAlertState, fromPhoneNumber, incidentCategoryKey)
-      const respondedByPhoneNumber = await this.alertSessionChangedCallback(changedAlertSession)
+      const { respondedByPhoneNumber, replaceReturnMessageToRespondedByPhoneNumber, replaceReturnMessageToOtherResponderPhoneNumbers } =
```

Comment:
Once you've got this figured out in a way that works and you like it, remember to update the description in the `README`. 

This will also require a change in Buttons for it to keep working.